app/templates/application.hbs

```diff
@@ -14,7 +14,7 @@
         </div>
     </div>
 </nav>
+{{pinned-notifications}}
 <div id="content">
```

Comment:
<div id="content" {{if visible "visible-class" "not-visible-class"}}

if you don't want to add a class when it is not visible, you can omit "not-visible-class"